doc/developer-guide/internal-libraries/Extendible.en.rst

```diff
@@ -80,18 +80,19 @@ the type's constructor, destructor, and serializer. And to avoid corruption, the
    }
 
 
-When an derived class is instantiated, :func:`template<> alloc()` will allocate a block of memory for the derived class and all added
-fields. The only memory overhead per instance is an uint16 used as a offset to the start of the extendible block.
+When an derived class is instantiated, :func:`template<> create()` will allocate a block of memory for the derived class and all added
```

Comment:
I wonder if this should be `make_shared` or even a specialization of `std::make_shared`.